install.sh

```diff
@@ -240,7 +240,7 @@ nvm_check_global_modules() {
   MODULE_COUNT="$(
     command printf %s\\n "$NPM_GLOBAL_MODULES" |
     command sed -ne '1!p' |                     # Remove the first line
-    wc -l | tr -d ' '                           # Count entries
+    command wc -l | tr -d ' '                           # Count entries
```

Comment:
let's revert this since it hasn't been a problem before (note that `tr` isn't prefixed either)